mava/configs/env/scenario/clean-20x20x20a.yaml

```
@@ -0,0 +1,8 @@
+# The config of the 20x20x40a scenario
+name: Cleaner-v0
+task_name: clean-20x20x40a
+
+task_config:
+  num_rows: 20
+  num_cols: 20
+  num_agents: 40
```

Think this is a typo
```suggestion
  num_agents: 20
```